compiler/rustc_mir_build/src/build/expr/into.rs

```diff
@@ -149,7 +149,11 @@ impl<'a, 'tcx> Builder<'a, 'tcx> {
             }
             ExprKind::LogicalOp { op, lhs, rhs } => {
                 let condition_scope = this.local_scope();
-                let source_info = this.source_info(expr.span);
+                let expr_span = expr.span;
+                let source_info = this.source_info(expr_span);
+
+                this.visit_coverage_branch_operation(op, expr_span);
+
```

Comment:
Nitpick: I don't think `expr_span` adds value here, compared to just writing `expr.span` twice.